test/test_mps.py

```diff
@@ -1300,6 +1301,20 @@ def test_to(self):
         self.assertEqual(torch.tensor(-8.34, device='cpu').to('mps', torch.int),
                          torch.tensor(-8.34, device='cpu').to('mps').to(torch.int))
 
+    @dtypes(torch.int32, torch.float32, torch.int64, device_type="mps")
```

Comment:
@mruberry do you know if `dtypes` decorator is deprecated or is it still supported? For some reason it does not appear in discover tests